docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
+
+[[enrolled-agent-not-showing-up]]
+=== I ran the Agent enrollment step but I don’t see the Agent in the UI
+If Elastic Agent was successfully enrolled, but doesn't show up in the Fleet list a common problem is that it was not 'started'.  Starting the Agent is a separate step which can be done as follows as the below.
+
+On linux & macos hosts:
+./elastic-agent run
+
+On Windows hosts:
+elastic-agent.exe run
+
+[[where-are-the-agent-logs]]
+=== I need to find the logs when the Agent starts up
+
+When started successfully, the logs are in the folder where the Agent was started, in a sub-directory 'data/logs/metricbeat'
+If that path of logs does not exist, it indicates Agent did not successfully run metricbeat, which is a higher level problem to triage.
+
+[[what-is-my-agent-config]]
+=== I need to know the configuration used in the running Agent.
+
+You can inspect the files in the folder from where you ran the Agent.  The elastic-agent.yml file will have an uncommented citation if it is in 'Fleet' mode:
+Management: mode: "fleet"
+
+The action_store.yml has the whole configuration, unencrypted.  Look for 'outputs:hosts' section with elasticsearch location.  This file will also have the current versions of the packages in use by the current configuration.  The version of the Agent is in the download folder / zip.
+
+[[where-is-the-data-agent-is-sending]]
+=== I'm not sure {ingest-manager} app is showing the data my Agent is sending.
+
+I think I have Agent set up and running and there are no errors on the Agent logs and I still don't see data in Kibana.  The recommended way to assess this is to go straight to Elasticsearch data queries, or maybe just use Kibana Discover app.  Look for any data in the index your Agent is sending to (metrics-* for example) and see if the name.host of your host machine shows.  If not, then perhaps the data is getting trapped / blocked in the network or isn't sent off of the Agent host correctly in the first place due to some Firewall or security reason.  While it should be redundant, it is all the same potentially helpful to install the stand-alone Metricbeat (see the Elastic docs for information) to see if that works to send data into ES, which would indicate a bug or problem in the Agent side.
+
+[[i-deleted-my-agent]]
+=== I deleted my agent in the Fleet {ingest-manager} app so now what?
+
+Its ok, be cool: the data is still in Elasticsearch.  But to get the Agent back in the UI, you will need to stop, then re-enroll the Agent on the host and run it again.
```

Comment:
"Its ok, be cool" might not work for some folks (it's kind of like saying calm down). :-) Maybe just say, "That's OK. The data is...."

Reply:
Not sure if this should be in troubleshooting. If we leave it in this section, we need to describe the problem that users encounter and how to solve it. Is the problem that they deleted it from Fleet and can't figure out how to make it show up again?

I guess the other option is to make this a general FAQ. But I do think that some of the info covered here should be in main docs.

Reply:
yeah, I purposefully tried to put something light-hearted here, I'll tone it down.  Its a fine line to walk, thanks for the note.

I think I'd like to leave this in the spirit of a combined FAQ and Troubleshooting doc.  When we split them we can move content around for sure.  I could re-phrase things in the form of a question, but this seemed straight forward and transparent and I'm don't see the value at present time, we can iterate on it later right?